src/wagtail_ai/static_src/image_description/main.css

```diff
@@ -0,0 +1,108 @@
+.wagtailai-input-wrapper {
+  display: flex;
+  align-items: stretch;
+  gap: 0.3125rem;
+}
+
+.wagtailai-button {
```

Comment:
**suggestion:** We could use Wagtail's `button` class here for consistency, and because this white background looks a bit out of place in dark mode?
<img width="579" alt="Screenshot 2024-03-11 at 13 35 15" src="https://github.com/wagtail/wagtail-ai/assets/27112/ece93dcf-02ff-4bfc-a9d4-1ad75791d774">

Reply:
Cool! How did you get the wand icon to show as white? I've removed all styles from my custom class and just added `button`; the background looks right, but the icon is black, which doesn't give enough contrast.

<img width="278" alt="Screenshot 2024-03-13 at 11 01 44" src="https://github.com/wagtail/wagtail-ai/assets/27617/8c594b84-cb2b-4c41-84f9-fe6874659beb">

Reply:
Ah yes sorry I forgot I did that. I removed all the `fill` attributes from `wand_icon.svg`, the custom CSS for the button was reduced to:

```css
.wagtailai-button > svg {
    fill: white;
}
```

and then I made sure both classes were added to the button:

```javascript
button.classList.add("button", "wagtailai-button")
```

Reply:
👌 thanks! I've also adjusted the animation a bit to match the new inactive button background colour.